src/arrayMethodJoin.js

```diff
@@ -1,11 +1,28 @@
 'use strict';
 
-/**
- * Implement method join
- */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
use default parametr

Reply:
the default param is still not fixed. please set the default value for `separator`